Website/Packages/datax-pipeline/src/modules/flowDefinition/components/schedule/scheduleSettingsContent.jsx

```diff
@@ -0,0 +1,329 @@
+// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as Models from '../../flowModels';
+import { DetailsList, DetailsListLayoutMode, Selection, SelectionMode, CheckboxVisibility, DefaultButton } from 'office-ui-fabric-react';
+import { Colors, IconButtonStyles, PanelHeader, PanelHeaderButtons, ScrollableContentPane, StatementBox } from 'datax-common';
+import RecurringScheduleSettings from './recurringScheduleSettings';
+import OneTimeScheduleSettings from './oneTimeScheduleSettings';
+import * as Styles from '../../../../common/styles';
+
+const batchColumns = [
+    {
+        key: 'columnBatchName',
+        name: 'Alias',
+        fieldName: 'id',
+        isResizable: true
+    },
+    {
+        key: 'columnBatchType',
+        name: 'Type',
+        fieldName: 'typeDisplay',
+        isResizable: true
+    },
+    {
+        key: 'columnBatchStatus',
+        name: 'Status',
+        fieldName: 'status',
+        isResizable: true
+    }
+];
+
+export default class scheduleSettingsContent extends React.Component {
```

Comment:
you should always capitolize the first letter of React components.

Reply:
fixed.